### ISSUE TITLE
gradlew check doesn't detect mis-formatted JSON files

### ISSUE THREAD

**Issue by mhalbritter** [oracle/graalvm-reachability-metadata#553]
1. `./gradlew scaffold --coordinates com.example:my-library:1.0.0`
2. Remove `testImplementation "com.example:my-library:$libraryVersion"` from `build.gradle`
3. Edit `metadata/com.example/my-library/1.0.0/index.json` to look like this:

```json
["jni-config.json","proxy-config.json","reflect-config.json","resource-config.json","serialization-config.json"]
```

4. Run `./gradlew check`

Expected: It should fail because the JSON is not formatted

Actual: Passes

**Comment by vjovanov** [comment]
@mhalbritter thanks for reporting! Is this issue blocking you?

**Comment by mhalbritter** [comment]
Nope, all good on my end. But until this is fixed, stuff on CI passes (meaning the PR will be green) even if the JSON is not well formatted - just something to keep an eye on.
